docs/scos/dev/upgradability-services/upgradability-guidelines.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Upgradability guidelines
+description: Reference information for evaluator and upgrader tools.
+last_updated: Nov 25, 2021
+template: concept-topic-template
+---
+
+## Entity name is not unique
+
+The names of the following entities should be unique to the extent of making it impossible to accidentally match the name of a core entity introduced in future:
+
+* Transfers
+* Transfer properties
+* Database tables
+* Database columns
+* Methods
+* Constants
+
+If a minor or major release introduces an entity with the same name, the entity in your project might change behavior or cause issues.
+
+### Making entity names unique
+
+To avoid unexpected issues and achieve the same result, make the names of entities unique. This section contains examples of transfer, database table, and method names. You can apply the same solution to any other entity.
+
+#### Examples of code and related errors
```

Comment:
Suggestion:

Consider restructuring the document in the following way:

## Dealing with the transfer names
- Description of the issue
- Output error 
- How the error is fixed (Renamed transfer name)

## Dealing with table names....

Reply:
I had to sacrifice the structure you suggest for avoiding h5. Also, this is a dev doc, and both the section with errors and the solution sections provide examples, not concrete steps to resolve the errors. 

However, while writing the doc, I was thinking about hiding all the examples into drop-downs to simplify comparing the error and solution snippets. What do you think?